packages/config/src/layer2s/zksyncera.ts

```diff
@@ -21,15 +21,14 @@ export const zksyncera: Layer2 = {
   display: {
     name: 'zkSync Era',
     slug: 'zksync-era',
-    warning: 'Currently only registered addresses can bridge to zkSync Era.',
```

Comment:
Is this true? I'm trying to bridge to zksync era using https://bridge.zksync.io/ and I keep getting this prompt 
<img width="480" alt="Screenshot 2023-03-23 at 11 59 26" src="https://user-images.githubusercontent.com/38423054/227183241-ac17b159-ab3a-488d-8acc-14b268d81cac.png">

Reply:
@michalsidzej this PR is supposed to go live tomorrow :) - Then, it will be true

Reply:
Cool, make sure you include it into description or mark the PR as draft :) Thanks for contributing tho 👍

Reply:
good point... added now, and will change tomorrow